test/integration/replace_or_add/controls/replace_or_add_add_a_line_matching_pattern.rb

```diff
@@ -0,0 +1,18 @@
+title 'replace_or_add Add a line exactly matching the pattern, pattern does not match the file'
```

Comment:
You can just wrap this in a control and get rid of the dummy control file. 

It'll also make the output slightly easier to read.